src/vs/workbench/contrib/notebook/browser/contrib/outline/notebookOutline.ts

```diff
@@ -741,7 +741,7 @@ export class NotebookCellOutline implements IOutline<OutlineEntry> {
 			...options,
 			override: this._editor.input?.editorId,
 			cellRevealType: CellRevealType.NearTopIfOutsideViewport,
-			selection: entry.position,
+			selection: entry.range,
```

Comment:
this range is not fully correct. Here is what happens right now with your change. Given the following code in a javascript cell:
```js
// fn cell

const temp = "hello world"
console.log(temp);

function add(a, b) {
    return a + b;
}

console.log(add(1, 2));

function sub(a, b) {
    return a - b;
}

```

clicking the notebook outline entry for either add or sub should highlight the entire fn, and then clicking the entry for temp should highlight `temp = "hello world"`. This makes me think we may not be storing entry ranges correctly.